cloudigrade/config/settings/base.py

```diff
@@ -249,6 +249,8 @@
         {'queue': 'scale_up_inspection_cluster'},
     'account.tasks.run_inspection_cluster':
         {'queue': 'run_inspection_cluster'},
+    'account.tasks.persist_inspection_cluster_results_task':
+        {'queue': 'persist_inspection_task'},
```

Comment:
So that we're consistent with all the naming pattern, let's change the queue name to match the function name `persist_inspection_cluster_results_task`.

Reply:
…and this will require another change to shiftigrade since you already merged that.

Reply:
Pushed a change.  Thanks.